.devcontainer/postCreateCommand.sh

```diff
@@ -1,4 +1,5 @@
 # Install mosquitto client
+sudo apt-add-repository ppa:mosquitto-dev/mosquitto-ppa
```

Comment:
why we need the ppa? isn't the mosquitto latest in the default ubuntu apt repo?

Reply:
It's not :( the newer version isn't in the default apt, so we need the one that has support for using os certs and mqtt 5